docs/scos/dev/glue-api-guides/202307.0/dynamic-data-api/how-to-guides/how-to-configure-dynamic-data-api.md

```diff
@@ -10,7 +10,7 @@ redirect_from:
 This guide shows how to configure the Dynamic Data API endpoints.
 
 In order to incorporate a new endpoint for interacting with entities in the database, 
-it is necessary to add a corresponding row to the `spy_dynamic_entity_configuration` table. 
+it is necessary to add a corresponding row to the `spy_dynamic_entity_configuration` table or needs to create a configuration file and install configuration data, see ([Dynamic Data Install Configuration](/docs/scos/dev/feature-integration-guides/{{page.version}}/glue-api/dynamic-data-api-integration.html#dynamic-data-install-configuration))
```

Comment:
The section "dynamic-data-install-configuration" does not exist, so the link to the section does not work; it simply opens the Dynamic Data API integration guide.
Therefore, please, adjust the link and it's name accordingly.
```suggestion
add a corresponding row to the `spy_dynamic_entity_configuration` table or create a configuration file and install configuration data. For details, see [Dynamic Data Install Configuration](/docs/scos/dev/feature-integration-guides/{{page.version}}/glue-api/dynamic-data-api-integration.html#dynamic-data-install-configuration)
```